edxval/api.py

```diff
@@ -478,34 +483,53 @@ def copy_course_videos(source_course_id, destination_course_id):
     if source_course_id == destination_course_id:
         return
 
-    videos = Video.objects.filter(courses__course_id=unicode(source_course_id))
+    course_videos = CourseVideo.objects.select_related('video', 'video_image').filter(
+        course_id=unicode(source_course_id)
+    )
 
-    for video in videos:
-        CourseVideo.objects.get_or_create(
-            video=video,
+    for course_video in course_videos:
+        destination_course_video, __ = CourseVideo.objects.get_or_create(
+            video=course_video.video,
             course_id=destination_course_id
         )
+        try:
+            VideoImage.create_or_update(
+                course_video=destination_course_video,
+                file_name=course_video.video_image.image.name
```

Comment:
more readable if we check hasattr(course_video, 'video_image') rather than catching does not exist.